### ISSUE TITLE
make-mingwlibs error

### ISSUE THREAD

**Issue by Rihy** [xbmc/xbmc#14664]
### Describe the bug
Here is a clear and concise description of what the problem is:

I advanced while looking at the build guide, but I will fail.
make-mingwlibs.bat
```
~~~~~
patching file configure
Reversed (or previously applied) patch detected!  Skipping patch.
1 out of 1 hunk ignored
ffmpeg-4.0.2-Leia-Alpha3 (x64) ............................... [Configuring]
ERROR: openssl not found

```
I can not build from here

Please help.

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Raspberri-Pi
 - [x] Windows
 - [ ] Windows UWP

 - Operating system version/name:WIN10 PRO 1803
 - Kodi version:

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by Rihy** [comment]
Step4 download-msys2.bat
So far I have done all.

download-dependencies.bat
It has already been executed.

**Comment by Rechi** [comment]
Have you also looked at the tip mentioned there?

> TIP: Look for the All formed packages ready! success message. If you see the message ERROR: Not all formed packages are ready!, execute the command again until you see the success message.

**Comment by Rihy** [comment]
All formed packages ready!
Of course we have also checked this display.

**Comment by Rechi** [comment]
Which exact `download-dependencies.bat`, `download-msys2.bat` and ` make-mingwlibs.bat` scripts have you executed? There are 6 of each one.

**Comment by Rihy** [comment]
6? Is not it three?

**Comment by pkerling** [comment]
This bug will be closed in the next few days if no further response is received.

**Comment by ncarthy** [comment]
I think I know what happened to Rihy because I had the same problem while trying to compile on x64. The instructions say to cd into ```%userprofile%\kodi\tools\buildsteps\windows\x64``` and then run download-dependencies.bat. However after that script has run it leaves me in the directory ```%userprofile%\kodi\tools\buildsteps\windows``` which is the parent directory, not the x64 directory. 

If you then run either ```download-msys2.bat``` or ```make-mingwlibs.bat``` without changing back down to the x64 directory it causes problems because those scripts will run as win32 instead of x64.

TL;DR after you run download-dependancies.bat make sure you are still in the x64 directory and change back to that directory if you are not!

**Comment by garbear** [comment]
Good observation, now I remember the same happening to me as well.

**Comment by KostasAndrianos** [comment]
I just spent a good hour trying to figure out what was going wrong with my environment and setup. I'll try to fix this soon, it drove me nuts
